benchmark/misc/bench-hrtime.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const ITER = 1e6;
+
+var t = process.hrtime();
+
+for (var i = 0; i < ITER; i++) {
+  process.hrtime(t);
+}
+
+t = process.hrtime(t);
+console.log(((t[0] * 1e9 + t[1]) / ITER).toFixed(1) + ' ns/op');
```

Comment:
I feel like `common.createBenchmark()` is a fail for benchmarking operations that finish in microseconds or less because at that point we're hitting the lower limit of how many CPU instructions it may take to perform an operation. Displaying the amount of time per operation is more insightful into what's actually going on, IMHO. But to stay consistent I can move the benchmarks over to `common`. That won't ruffle my feathers.